common/types/provider.go

```diff
@@ -115,10 +115,7 @@ func (p *protoTypeRegistry) FindFieldType(messageType string,
 	}
 	field, found := msgType.FieldByName(fieldName)
 	if !found {
-		field, found = p.pbdb.DescribeExtension(messageType, fieldName)
-		if !found {
-			return nil, false
-		}
```

Comment:
Could you help me understand this change? I would've thought that we want to perform the extension lookup here.

Reply:
The field lookup now falls back to looking at extensions, so the use of `DescribeExtension` isn't necessary here.